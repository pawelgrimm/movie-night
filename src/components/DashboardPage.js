import React from "react";
import MovieSearch from "./MovieSearch";
import MovieList from "./MovieList";
import VideoModal from "./VideoModal";

export const DashboardPage = ({ appElement }) => {
  return (
    <div className="content-container">
      <MovieSearch numResults={5} />
      <MovieList />
      <VideoModal appElement={appElement} />
    </div>
  );
};

export default DashboardPage;
