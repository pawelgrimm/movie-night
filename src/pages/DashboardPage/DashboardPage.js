import React from "react";
import AppPage from "../../components/AppPage/AppPage";
import DashboardInfoCard from "./components/DashboardInfo";
import ButtonGroup from "../../components/ButtonGroup/ButtonGroup";
import EndVotingButton from "./components/EndVotingButton";
import EditBallotButton from "./components/EditBallotButton";
import ResultsMovieListContainer from "../../containers/Results/components/ResultsMovieListContainer/ResultsMovieListContainer";
import ResultsProvider from "../../containers/Results/ResultsProvider";
import InProgBallotPageContainer from "../../containers/Results/components/InProgBallotPageContainer";

const DashboardPage = () => {
  return (
    <ResultsProvider>
      <AppPage>
        <InProgBallotPageContainer>
          <DashboardInfoCard />
          <ButtonGroup>
            <EndVotingButton />
            <EditBallotButton />
          </ButtonGroup>
          <ResultsMovieListContainer />
        </InProgBallotPageContainer>
      </AppPage>
    </ResultsProvider>
  );
};

export default DashboardPage;
