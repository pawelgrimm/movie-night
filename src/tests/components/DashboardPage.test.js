import { expect } from "@jest/globals";
import React from "react";
import { shallow } from "enzyme";
import DashboardPage from "../../components/DashboardPage";

test("should render ExpenseDashboardPage", () => {
  const wrapper = shallow(<DashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
