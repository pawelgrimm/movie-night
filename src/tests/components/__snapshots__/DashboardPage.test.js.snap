// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DashboardPage 1`] = `
<div
  className="content-container"
>
  <MovieSearch
    numResults={5}
  />
  <Connect(MovieList) />
  <Connect(VideoModal) />
</div>
`;
